src/future.js

```diff
@@ -59,7 +59,6 @@ export function isFuture(x){
   return x instanceof Future || type(x) === $$type;
 }
 
-Future['@@type'] = $$type;
```

Comment:
The update to sanctuary-type-identifiers 3 makes it so the removed line above would cause `Future` itself to be identified as a member of `$$type`, as we've also seen in recently https://github.com/fluture-js/fluture-sanctuary-types/pull/67/commits/d8c5527de0d3cdaffc8c47d106d5499290ae581f.

As a result, I am forced to remove this property. I don't want to remove this property, because it means that Fluture no longer supports older code that uses sanctuary-type-identifiers@2 (such as older versions of Fluture, who now lose the ability to understand that Fluture version 12 exposes Future instances.

This is the reason I have been advocating for putting a version number in the property name, eg:

```js
Future['@@type@1'] = 'fluture/Future@3'
Future['@@type@2'] = 'fluture/Future@3'
Future.prototype['@@type@3'] = 'fluture/Future@3'
```

This way, support for multiple versions of a spec can live peacefully alongside each other. I have also advocated for this kind of thing in Fantasy Land, but it's never well received.

I am of the strong opinion that a *spec* must never become outdated. This means that if you make changes to a spec, you must do so in a way that doesn't interfere with the previous version of that spec, which should be treated as a completely different spec.